diff --git a/packages/desktop/src-tauri/src/windows.rs b/packages/desktop/src-tauri/src/windows.rs
index f361cbe38..049b54644 100644
--- a/packages/desktop/src-tauri/src/windows.rs
+++ b/packages/desktop/src-tauri/src/windows.rs
@@ -53,7 +53,7 @@ impl MainWindow {
             app,
             decorations,
         )
-        .title("OpenCode")
+        .title("O-CLI")
         .disable_drag_drop_handler()
         .zoom_hotkeys_enabled(false)
         .visible(true)
